FlagsmithClient/src/test/java/com/flagsmith/FeatureFlagTests.kt

```diff
@@ -25,7 +25,8 @@ class FeatureFlagTests {
         flagsmith = Flagsmith(
             environmentKey = "",
             baseUrl = "http://localhost:${mockServer.localPort}",
-            enableAnalytics = false
+            enableAnalytics = false,
+            enableCache = false
```

Comment:
If we set the default to false as per my comment above, I don't think we'll need this change.